spacemesh/v1/smesher.proto

```diff
@@ -39,6 +39,14 @@ service SmesherService {
         option (google.api.http) = {
             post: "/v1/smesher/smesherid"
         };
+        option deprecated = true; // will return empty response in v1.4.x
```

Comment:
Wouldn't it generate a compiler warning when you try to implement `SmesherID` service in go-sm?

Reply:
Afaik only on the client side - we do implement the deprecated handler (otherwise we wouldn't satisfy an interface) and the linter doesn't complain.